src/views/table_element_footer.ts

```diff
@@ -0,0 +1,38 @@
+/*!
+ * © 2019 Atypon Systems LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ManuscriptNodeView } from '@manuscripts/transform'
+
+import { BaseNodeProps, BaseNodeView } from './base_node_view'
+import { createNodeView } from './creators'
+
+export class TableElementFooterView<PropsType extends BaseNodeProps>
+  extends BaseNodeView<PropsType>
+  implements ManuscriptNodeView
+{
+  public initialise = () => {
+    this.createDOM()
+  }
+
+  public createDOM = () => {
+    this.dom = document.createElement('div')
+    this.dom.classList.add('table-footer')
+    this.dom.setAttribute('id', this.node.attrs.id)
+    this.contentDOM = this.dom
```

Comment:
As I mentioned on IM, I think this is a bit unintuitive. Both of those properties have certain usage as public elements and if we don't want prosemirror to manage contenDOM for us we should just not assign it at all and use this.dom only. What do you think?

Reply:
Yeah, it might seem redundant to explicitly set `this.contentDOM = this.dom`. It's expected from ProseMirror to consider `this.dom` as the `content DOM` automatically. But without that line, the content disappears so I had to do that to ensure that ProseMirror recognizes the main DOM as the content.

Does that make sense to you?

Reply:
I have no problem with that; it will work, but the DOM will have an extra `div` wrapper.

Something like this:
![Screenshot 2024-01-18 at 8 35 50 PM](https://github.com/Atypon-OpenSource/manuscripts-body-editor/assets/33928809/af2f9cc2-7ab1-4c24-93a2-4370e7421e71)

Reply:
Done!